change/@ni-nimble-components-a5b4c275-28d7-4509-af45-748fdd2f69fb.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@fredvisser do we want to modify the matrix stories to specify width and avoid the visual diff or accept the changes with the new min-width? I don't think I feel strongly either way (but haven't look through the changes, there are a bunch)

Reply:
Actually scanning a couple things like this are making me a bit nervous about the stability of these:
![image](https://github.com/ni/nimble/assets/1588923/5189b3f3-d29f-4924-8519-df71d921e84c)